Dockerfile

```diff
@@ -15,7 +15,7 @@ RUN mkdir /etc/service/cromwell && \
     apt-key adv --keyserver hkp://p80.pool.sks-keyservers.net:80 --recv-keys 58118E89F3A912897C070ADBF76221572C52609D && \
     echo 'deb https://apt.dockerproject.org/repo ubuntu-trusty main' > /etc/apt/sources.list.d/docker.list && \
     apt-get update && \
-    apt-get install yq docker-engine=1.8.2-0~trusty && \
+    apt-get -yq install docker-engine=1.8.2-0~trusty && \
```

Comment:
I'm assuming your Hmm is the equivalent a ToL without the out loud? 😉  Our Dockerfile may need other features to improve it, via other PRs, but at the moment the "install yq" is causing false negative build failures in jenkins.